docs/modules/ROOT/pages/dedicated-cluster.adoc

```diff
@@ -1,9 +1,24 @@
 = Dedicated Clusters
-:description: {hazelcast-cloud} Dedicated gives you fine-grained control over the hardware and resources available to your cluster. It's your responsibility to scale your clusters when you need more resources. Dedicated clusters also offer strict container isolation for projects that require enhanced data security.
+:description: {hazelcast-cloud} Dedicated is a fully-managed cloud service that allows you to maintain control over the hardware, resources, and deployment options that are available to your cluster. Dedicated means you get specific servers (cloud instances) all to yourself. 
 :page-dedicated: true
 
 {description}
 
+You can design a Hazelcast topology that meets your specific compliance and service design needs.
+
+[discrete] 
+==== Data and Security Regulations
+- Deploy your clusters to specific regions to comply with data residency laws.
+- Benefit from strict container isolation for projects that require enhanced security. {hazelcast-cloud} Dedicated is SOC 2 and ISO 27001 compliant.
+
+[discrete] 
+==== Service Design
+- Specify the memory capacity and number of nodes for all provisioned clusters. You decide when to scale your clusters if they need more resources.  
```

Comment:
```suggestion
** Specify the memory capacity and the number of nodes for all provisioned clusters. You decide when to scale your clusters if they need more resources.  
```

Need to check if 'nodes' refers to Kubernetes nodes or Hazelcast members.

Reply:
@kubamarchwicki. Could you confirm the details above?

Reply:
Yes. Hazelcast has members - cluster even brags about it in the logs

```
Members {size:1, ver:1} [
	Member [10.25.131.35]:5701 - bb968318-fec7-4b61-a962-7579a92d9f97 this
]
```

Reply:
Added explanation to make it clearer what we mean by scaling in and out.